ext/cfx-ui/src/assets/languages/locale-de.json

```
@@ -44,22 +51,35 @@
     "#Settings_MenuAudioDesc": "Aktiviere Hintergrundmusik während Inaktivität im Hauptmenü",
     "#Settings_GameStreamProgress": "Streaming-Fortschritt",
     "#Settings_GameStreamProgressDesc": "Zeigt den Fortschritt beim Herunterladen von im Spiel gestreamten Inhalten an. (BETA™)",
-    "#DirectConnect_TimedOut": "Zeitüberschreitung der Serveranfrage.",
+    "#Settings_QueriesPerMinute": "Serversuche: Maximale Pings / Minute",
+    "#Settings_CustomEmoji": "Eigenes FiveM Wasserzeichen",
+    "#Settings_CustomEmojiDesc": "Ein Emoji für das Wasserzeichen im Spiel.",
+    "#Settings_CustomEmojiUpsell": "Nicht vefügbar! Spende $5/Monat oder mehr, um einen offiziellen FiveM-Patreon Account zu bekommen und Dir Dein eigenes Emoji als Wasserzeichen zu setzen.",
+    "#Settings_ConnectedProfiles": "Verbundene Identitäten",
+    "#SettingsCat_Connection": "Verbindung",
+    "#SettingsCat_Interface": "Benutzeroberfläche",
+    "#SettingsCat_Game": "Spiel",
+    "#SettingsCat_Account": "Benuterkonto",
+    "#DirectConnect_TimedOut": "Zeitüberschreitung bei Serverabfrage.",
     "#DirectConnect_Failed": "Server konnte nicht abgefragt werden.",
-    "#DirectConnect_ServerInvalid": "Server konnte nicht gefunden werden",
-    "#DirectConnect_ServerInvalid_SubTitle": "Bitte überprüfe, ob du eine korrekte Adresse eingegeben hast.",
+    "#DirectConnect_Loading": "Lädt...",
+    "#DirectConnect_ServerInvalid": "Server kann nicht gefunden werden",
+    "#DirectConnect_ServerInvalid_SubTitle": "Stellen Sie sicher, dass Sie die richtige Adresse eingegeben haben.",
     "#DirectConnect_IPPort": "IP:Port",
     "#DirectConnect_Connect": "Verbinden",
     "#DirectConnect_LastPlayed": "Zuletzt gespielt",
     "#ServerDetail_Players": "Spieler",
-    "#ServerDetail_PlayersEmpty": "Es sind keine Spieler auf diesem Server online.",
+    "#ServerDetail_PlayersEmpty": "Auf diesem Server sind keine Spieler aktiv",
```

"Aktiv" sounds as if no one is online on this server at any time. I would stick with "Es sind keine Spieler auf diesem Server online."

```suggestion
    "#ServerDetail_PlayersEmpty": "Es sind keine Spieler auf diesem Server online.",
```